README.md

```diff
@@ -3,18 +3,104 @@
 This application allows users to create and manage work orders.
 
 # Setup
-## Dev environment
+## Database
 To create the databases for the dev environment: `rake db:setup`
 
+## Broker
+To create the exchanges, queues and usernames etc. use the GitLab repo: [aker-environments](https://gitlab.internal.sanger.ac.uk/aker/aker-environments)
+
 # Testing
 To run ALL the tests for this project: `bundle exec rake`
 
 To run just the rspec tests: `bundle exec rspec`
 
-Note that if any changes are made in the `app/client`,`app/assets` or `app/javascript` directories, any files in the `public/assets`,`public/packs` or `public/packs-test`  directory must be removed, and webpack must be run with the `webpack.production.js` configuration file:
+# Misc.
+## Assets
+Note that if any changes are made in the `app/client`,`app/assets` or `app/javascript` directories,
+any files in the `public/assets`,`public/packs` or `public/packs-test` directory must be removed,
+and webpack must be run with the `webpack.production.js` configuration file:
 
     rm -r public/assets
     rm -r public/packs
     rm -r public/packs-test
     RAILS_ENV=test bundle exec rake assets:precompile
-    RAILS_ENV=production bundle exec rake assets:precompile
\ No newline at end of file
+    RAILS_ENV=production bundle exec rake assets:precompile
+
+## Messages
+## Catalogue message
+The following message can be used to create a catalogue but currently process names have to be
+unique so adjust the message as required.
+```json
```

Comment:
This could be replaced with the larger catalog (pending code-review)

Reply:
True, it would be helpful to keep this message up-to-date.